### PR TITLE
fix(docs): dangled `let` modifier for a `var` variable

### PR DESCRIPTION
**Pull request by flyingImer**
*By submitting this pull request, I confirm that my contribution is made under the terms of the 
[Monada Contribution License](https://docs.winglang.io/terms-and-policies/contribution-license.html)*.

### CODE REVIEW

File: docs/04-reference/winglang-spec.md [hunk 1]
```diff
@@ -406,7 +406,7 @@ Variables can be reassigned to by adding the `var` modifier:
 
 ```ts
 // wing
-let var sum = 0;
+var sum = 0;
```
Comment (eladb) [inline docs/04-reference/winglang-spec.md:409]: nope, our syntax is `let var` :-)

**Comment by eladb** [comment]
@flyingImer our syntax is `let var` and not just `var`. I am actually just writing a post explaining the reasoning behind it. Stay tuned.

I think this PR can be closed.

File: docs/04-reference/winglang-spec.md (hunk 1 above)
Comment (flyingImer) [reply docs/04-reference/winglang-spec.md:409]: oh was not aware of it, tried to look for reference in the doc, but no luck. probably I missed it somewhere, any pointer?

Comment (flyingImer) [reply docs/04-reference/winglang-spec.md:409]: `Let let be let. (Elad B. 2022)` I figured

**Pull request closed without merging**